compose.prod.yaml

```diff
@@ -82,6 +82,7 @@ x-airflow-common:
     # The following line can be used to set a custom config file, stored in the local config folder
     # If you want to use it, outcomment it and replace airflow.cfg with the name of your config file
     # AIRFLOW_CONFIG: '/opt/airflow/config/airflow.cfg'
+    AIRFLOW_CONN_GOOGLE_CLOUD_DEFAULT: 'google-cloud-platform://?key_path=%2Fopt%2Fairflow%2Fconfig%2Fsul-rialto-google-drive-key.json&scope=https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fdrive%2Chttps%3A%2F%2Fwww.googleapis.com%2Fauth%2Fspreadsheets&project=sul-rialto&num_retries=5'
```

Comment:
this creates the connection in airflow, specifying the location of the service account JSON file

this file will be placed on the VMs by puppet/vault combo, for localhost devs will need to get it and place it in the right spot as described in the README